### ISSUE TITLE
Scheduling Blocks that failed to restore

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#279]
Each of these SBs appears to have some failure in the restoration, though some of them are marked as OK

- [ ] https://github.com/ACES-CMZ/reduction_ACES/issues/145
- [ ] https://github.com/ACES-CMZ/reduction_ACES/issues/114
- [ ] https://github.com/ACES-CMZ/reduction_ACES/issues/134
- [ ] https://github.com/ACES-CMZ/reduction_ACES/issues/132

These are the black fields in this image:
![image](https://user-images.githubusercontent.com/143715/195198485-573b9485-5c3f-47da-904d-fdd53a8242a8.png)
which (at least in some cases) indicates that the measurement set being imaged are not calibrated.

There may be a variety of underlying problems, but these seem to be fundamental and require either a helpdesk ticket or a re-download of the data.

**Comment by d-l-walker** [comment]
I'm a bit confused about #132 being in this list. I can see that in the image you've posted that it doesn't look right. But this is one that I reviewed and marked as 'Done', the images in this version of the weblog produced look OK to me? 

https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_ae_03_TM1_0/html/t2-4m.html?sidebar=sidebar_stage38&ms=all&subpage=t2-4m_details.html

The weblog for this run however looks bad, and the data are clearly uncalibrated: https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_ae_03_TM1_1/html/t2-4m.html?sidebar=sidebar_stage10&ms=all&subpage=t2-4m_details.html

So it seems that the run using PL version 6.4.3.2 didn't work as expected, but the one with version 6.2.1.7 looks fine?

**Comment by keflavich** [comment]
Yes, exactly, that's the problem: the pipeline run here failed for reasons unknown.
